src/js/components/stock-movement/outbound/StockMovementOutboundTable.jsx

```diff
@@ -207,6 +207,19 @@ const StockMovementOutboundTable = ({
       sortable: false,
       Cell: row => (<TableCell {...row} defaultValue="None" />),
     },
+    {
+      Header: <Translate id="react.stockMovement.request.approvers.label" defaultMessage="Approvers" />,
+      accessor: 'approvers',
+      minWidth: 250,
+      sortable: false,
+      Cell: row =>
+        (<TableCell
+          {...row}
+          tooltip
+          defaultValue="None"
+          value={row.value.map(catalog => catalog.name).join(', ')}
```

Comment:
I know it's probably copied :laughing: but please change it from `catalog` to `user`/`approver`

Reply:
Additionally, maybe we can merge those two methods into one? Each of them makes one iteration over values, it would be nice if we can reduce them to one iteration. I mean something like:
```
row.value?.reduce((acc, curr) => acc + catalog.name , "")
```
(wrote by hand, not sure if works)

Reply:
we need commas in between the items so your code would need to look at least like
```js
row.value?.reduce((acc, curr) => acc + `${catalog.name}, ` , "")
```
Additionally we are joining elements with a comma `, ` and in the code above we would always have commas at the end of the last element, and in case of a single element array it will also appear unless we do some kind of a `if(index is not lastElement)` logic.

My opinion is that we should not sacrifice code readability with trying to achieve peak performance, so I think it shouldn't be a big deal

Reply:
I forgot about the last element with comma, so it looks good to me